crypto/rand/rand_unix.c

```diff
@@ -12,6 +12,7 @@
 #include "internal/cryptlib.h"
 #include <openssl/rand.h>
 #include "rand_lcl.h"
+#include "internal/rand_int.h"
```

Comment:
So re the double forward declaration of `RAND_POOL`, if you just move this one line up one step...